packages/oas-to-har/src/index.js

```diff
@@ -40,13 +41,42 @@ function isPrimitive(val) {
   return typeof val === 'string' || typeof val === 'number' || typeof val === 'boolean';
 }
 
+function stringify(json) {
+  return JSON.stringify(removeUndefinedObjects(typeof json.RAW_BODY !== 'undefined' ? json.RAW_BODY : json));
+}
+
+function getNameFromDataUrlType(type) {
+  const properties = type.split(';').filter(param => {
+    return param.split('=')[0] === 'name';
+  });
+
+  return properties.length !== 1 ? 'unknown' : properties[0].split('=')[1];
```

Comment:
How does this line work? Wouldn't the ternary statement always evaluate to a Boolean value?